WcaOnRails/app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -170,13 +170,21 @@ def index_for_group_type
           }
         end
       end
+    elsif group_type == UserGroup.group_types[:translators]
+      roles.concat(TranslationsController.translators_to_roles)
+    else
+      render status: :unprocessable_entity, json: { error: "Invalid group type" }
     end
 
     # Filter the list based on the permissions of the logged in user.
     roles = filter_roles_for_logged_in_user(roles)
 
     # Filter the list based on the other parameters.
-    roles = filter_roles_for_parameters(roles: roles, status: params[:status], is_active: params[:isActive])
+    roles = filter_roles_for_parameters(
+      roles: roles,
+      status: params[:status],
+      is_active: params.key?(:isActive) ? ActiveRecord::Type::Boolean.new.cast(params.require(:isActive)) : nil,
```

Comment:
Consider outsourcing this into a separate line of code (or two)

Reply:
Done